src/models/CommonModel.ts

```diff
@@ -186,6 +189,7 @@ export class CommonModel extends CommonSchema<CommonModel> {
     const mergeFromAdditionalProperties = mergeFrom.additionalProperties;
     if (mergeFromAdditionalProperties !== undefined) {
       if (mergeToAdditionalProperties !== undefined) {
+        Logger.warn(`Found duplicate additionalProperties for model. additionalProperties from ${mergeFrom.$id || 'unknown'} merged into ${mergeTo.$id || 'unknown'}`, mergeTo, mergeFrom, originalSchema);
```

Comment:
info

Reply:
Others should be reconsidered too.

Reply:
see https://github.com/asyncapi/generator-model-sdk/pull/184#discussion_r632957284